lib/slimmer/processors/body_inserter.rb

```diff
@@ -10,7 +10,17 @@ def filter(src, dest)
       source_markup = src.at_css(@source_selector)
       destination_markup = dest.at_css(@destination_selector)
 
-      raise(Slimmer::SourceWrapperNotFoundError, "Source wrapper div not found", caller) if source_markup.nil? && wrapper_check?
+      if source_markup.nil? && wrapper_check?
+        raise(Slimmer::SourceWrapperNotFoundError, <<~END_STRING, caller)
+          Slimmer did not find the wrapper div in the source HTML.
+          The following cause can be safely ignored, other need
+          to be investigated. One possible cause is that the
+          requested format was different from HTML e.g. JavaScript,
+          but the action only supports the HTML template and layout.
+          Rails only applies layouts matching the requested format,
+          so the layout was not applied.
+        END_STRING
+      end
```

Comment:
I think you could probably cut this down a bit? 

```
Slimmer did not find the wrapper div in the source HTML.
This could be because the request was for a format
other than HTML (eg. JavaScript) which Slimmer does not
support.
```

Reply:
If we are leaving this in, do we still need the wrapper check? Where has that env var been set?

Reply:
I wonder, if the suggested message might not be a bit misleading. The issue is not with Slimmer really, but with Rails. Slimmer supports anything that contains a wrapper. The issue is with how Rails layouts work with respect to requested formats. What do you think?

Reply:
With respect to whether we still need `wrapper_check?`, it's up to you. It allows us to selectively enable it per app instead of applying it to all of them. It's set in govuk-helm-charts. But the solution is general, so it's not really necessary. Please, let me know what makes more sense to you?

Reply:
Is this accurate?
```
Slimmer did not find a div with ID "wrapper" in the source HTML.
This could be because a request was made to your Rails application
for a format that it does not support. For example a javascript request,
when the application only has HTML templates. 
```

Reply:
Yes, I think we can go with that message.

Reply:
I've applied the changes. I'll also prepare the PR to remove the env var.